man/systemd-boot.xml

```diff
@@ -154,6 +154,14 @@
         <listitem><para>Show systemd-boot, UEFI, and firmware versions</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><keycap>r</keycap></term>
+        <term><keycap>F5</keycap></term>
+        <listitem><para>Change screen resolution, skipping any unsupported modes. The new mode
+        will have to be manually changed in <citerefentry><refentrytitle>loader.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+        to persist across boots.</para></listitem>
```

Comment:
why not store in EFI var?

Reply:
I intend to add a save last booted entry for next time feature (again). I did this a long time ago using efi vars and that was rejected due to firmware fucking up with excessive stores to nvram.

So I was therefore gonna write those to the efi partition instead and leave the efi vars alone. Should I still make a consle mode var then?

Reply:
writing EFI vars is fine, as long as it's done "seldomly", i.e. on explicit user request. writing to them on each boot is a problem though. 

hence, i think this would be a natural thing to place in an efi var, because it's never touched except if the user explicitly tells us to.

we try to limit our fs write use, actually, since there are some doubts about the fs code, and that it might be even worse than the efi var code in the firmwares. so far we only to fs renames and block level wites (i.e. no writes that cause allocation of new space for files)